pkg/vsphere/session/session.go

```diff
@@ -125,7 +128,12 @@ func LimitConcurrency(rt http.RoundTripper, limit int) http.RoundTripper {
 }
 
 // NewSession creates a new Session struct.
-func NewSession(config *Config) *Session {
+func NewSession(config *Config, vchMoref ...*types.ManagedObjectReference) *Session {
```

Comment:
I thought we decided to utilize config to pass in the VCHFolder -- specifically during portlayer start up in `portlayer.go`

Reply:
I'd rather not bake more knowledge into the session...if it's going to be a cache it can be a dumb cache

Reply:
If not available when config is passed in..then I'd set it on the session during startup (`portlayer.go`)